crates/beerus-cli/tests/cli.rs

```diff
@@ -2867,28 +2868,28 @@ mod test {
         );
 
         let program = vec![];
-        let constructor = vec![ContractEntryPoint {
+        let constructor = vec![LegacyContractEntryPoint {
```

Comment:
These tests are using the legacy format so just changed to Legacy for now. We can update as part of test coverage efforts.